metering/configuring_metering/metering-common-config-options.adoc

```diff
@@ -70,7 +70,12 @@ spec:
 ----
 
 == Node selectors
-If you want to run the metering components on specific sets of nodes, then you can set `nodeSelectors` on each component to control where each component of metering is scheduled. The `node-selectors.yaml` file below provides an example of setting node selectors for each component.
+You can run the metering components on specific sets of nodes. Set the `nodeSelector` on a metering component to control where the component is scheduled. The `node-selectors.yaml` file below provides an example of setting node selectors for each component.
+
+[NOTE]
+====
+Add the `openshift.io/node-selector: ""` namespace annotation to the metering namespace YAML file before configuring specific node selectors for the operand Pods. Specify `""` as the annotation value.
```

Comment:
Is there a difference between "operator pods" and "operand pods"? If not I'd suggest going with "operator pods".

Reply:
> Is there a difference between "operator pods" and "operand pods"? If not I'd suggest going with "operator pods".

@timflannagan1 Can you confirm if this should be "operator" or "operand"? Thank you.

Reply:
Yeah, it would be `operand Pods` here. Operator maps to the metering-ansible-operator whereas operand maps to Presto coordinator/worker, Hive metastore/server, reporting-operator.